doc/devdocs/ast.rst

```diff
@@ -231,6 +233,8 @@ See especially :ref:`devdocs-locks` for important details on how to modify these
     may be put here (if ``jlcall_api == 2``), or it could be set to `nothing`
     to just indicate ``rettype`` is inferred
 
+``min-age`` / ``max-age`` - The range of world ages for which this method instance is valid.
+
```

Comment:
Would be great to add devdocs quickly summarizing how/when the world counter is updated, how min/max ages are updated, and how to temporarily run in different worlds.